source/config/configSpec.py

```diff
@@ -27,6 +27,7 @@
 	#possible log levels are DEBUG, IO, DEBUGWARNING, INFO
 	loggingLevel = string(default="INFO")
 	showWelcomeDialogAtStartup = boolean(default=true)
+	preventSystemLock = featureFlag(optionsEnum="BoolFlag", behaviorOfDefault="disabled")
```

Comment:
can this be enabled by default? This would be typical expected behaviour IMO. If there's negative feedback on alpha/beta we can turn it off easily.

Reply:
I have now enabled it with a warning in the changelog/user guide.